person.rb

```diff
@@ -1,17 +1,18 @@
 require_relative './nameable'
 require_relative './capitalize'
-require_relative './decorator'
+require_relative './rental'
 require_relative './trimmer'
 
 class Person < Nameable
   attr_accessor :name, :age, :id
-  attr_reader :parent_permission
+  attr_reader :parent_permission, :rentals
```

Comment:
- [ ] Person class suppose to have getters and setters for rentals not getters only, So Please fix that.